app/Http/Controllers/Auth/LoginController.php

```diff
@@ -50,9 +54,14 @@ public function showLoginForm()
             $apiKeys = ApiKey::select('nocaptcha_sitekey', 'captcha_secretCheck', 'msg91_auth_key', 'terms_url')->first();
             $analyticsTag = ChatScript::where('google_analytics', 1)->where('on_registration', 1)->value('google_analytics_tag');
             $location = getLocation();
+            $google_status = SocialLogin::where('type', 'google')->first()->status;
```

Comment:
Use SocialLogin::select('status')->where('type', 'google')->value('status') like this or create some custon static function in model mutator